cmd/internal-ip/main.go

```diff
@@ -1,63 +1,87 @@
 package main
 
 import (
+	"errors"
+	"flag"
 	"fmt"
 	"net"
 	"os"
 	"strings"
 )
 
+// Set display texts to defaults.
+var fullText = "unknown"
+var shortText = "unknown"
+
 func main() {
+	var iface = &net.Interface{}
+
+	// Check if interface parameter was passed in
+	var ifaceNameFlag = flag.String("ifaceName", "", "-ifaceName wlp2s0")
```

Comment:
```suggestion
	var ifaceNameFlag = flag.String("ifaceName", "", "Specify the name of the interface of which to display its IP.")
```

Reply:
The third parameter of any flag type is typically set to a help text (displayed when you run the program with the ```-help``` argument).